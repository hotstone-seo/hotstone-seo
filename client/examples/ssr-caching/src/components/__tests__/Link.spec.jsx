import React from 'react';
import Link from '../Link.jsx';

test('Should render Link correctly', () => {
  const tree = shallow(<Link to="/">Link</Link>);

  expect(tree).toMatchSnapshot();
});
