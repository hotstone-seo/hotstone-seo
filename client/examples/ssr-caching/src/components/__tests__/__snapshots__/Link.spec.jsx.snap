// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Link correctly 1`] = `
<BrowserRouter>
  <CustomLink
    to="/"
  >
    Link
  </CustomLink>
</BrowserRouter>
`;
